### PR TITLE
BAEL-4438 Add Postman Collections and Live test for Keycloack Endpoints

### PR DESCRIPTION
**Pull request by maryarm**

### CODE REVIEW

File: oauth-jwt/jwt-auth-server/src/test/resources/Keycloack.postman_collection.json [hunk 1]
```diff
@@ -0,0 +1,746 @@
+{
+    "variables": [],
+    "info": {
+        "name": "Keycloack",
```
Comment (KevinGilmore) [inline oauth-jwt/jwt-auth-server/src/test/resources/Keycloack.postman_collection.json:4]: note the correct spelling is "Keycloak" -- let's correct it in the JSON content and filenames

File: oauth-jwt/jwt-auth-server/src/test/resources/Keycloack.postman_environment.json [hunk 2]
```diff
@@ -0,0 +1,76 @@
+{
+  "id": "432f873b-56d9-94f8-8393-eda4e8223486",
+  "name": "Keycloack",
```
Comment (KevinGilmore) [inline oauth-jwt/jwt-auth-server/src/test/resources/Keycloack.postman_environment.json:3]: Keycloak

**Review by KevinGilmore — CHANGES_REQUESTED** [review]
Some of the filenames have "Keycloak" spelled incorrectly as "Keycloack" both in the filename and in the content.

File: oauth-jwt/jwt-auth-server/src/test/resources/Keycloack.postman_collection.json (hunk 1 above)
Comment (maryarm) [reply oauth-jwt/jwt-auth-server/src/test/resources/Keycloack.postman_collection.json]: Fixed.

File: oauth-jwt/jwt-auth-server/src/test/resources/Keycloack.postman_environment.json (hunk 2 above)
Comment (maryarm) [reply oauth-jwt/jwt-auth-server/src/test/resources/Keycloack.postman_environment.json]: Fixed.

**Pull request merged**